apps/common-app/src/examples/EmptyExample.tsx

```diff
@@ -1,19 +1,248 @@
-import { Text, StyleSheet, View } from 'react-native';
-
 import React from 'react';
```

Comment:
I will revert changes from this file before merging. I don't think we need to include this in examples.